extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -286,9 +286,13 @@ export async function isFunctionProject(folderPath: string): Promise<boolean> {
 
 /**
  * Prompts the user to select type of binding and returns result
+ * @param objectType (Optional) The type of object user choose to insert/upsert into
  * @param funcName (Optional) Name of the function to which we are adding the SQL Binding
+ * @returns binding type or undefined if the user cancelled out of the prompt
  */
-export async function promptForBindingType(funcName?: string): Promise<BindingType | undefined> {
+export async function promptForBindingType(objectType?: string, funcName?: string): Promise<BindingType | undefined> {
+	// check to see if objectType is view
+	let isView = (objectType === constants.sqlView || objectType === 'View');
```

Comment:
Instead of having this check I would suggest making an ObjectType enum and using that as the parameter. Much easier to use and avoids situations where you might miss checking both kinds.